packages/cloud-core/lib/scss/base/modal/ModalHeight/index.scss

```diff
@@ -2,14 +2,14 @@
 SPDX-License-Identifier: GPL-3.0-only */
 
 .modal-height {
-  width: 100%;
-  max-width: 600px;
-  max-height: 100%;
   border-radius: 1.5rem;
   position: relative;
   z-index: 9;
+  max-height: 100%;
+  width: 100%;
+  max-width: 600px;
```

Comment:
lets use `rem` instead of `px` and  please and `vh`, `vw` instead of `%`.
So far I think its `10px = 1rem`